python/ray/train/_internal/storage.py

```diff
@@ -471,17 +471,16 @@ def __init__(
         self._check_validation_file()
 
     def __str__(self):
-        attrs = [
-            "storage_path",
```

Comment:
What's the purpose of deleting this `self.storage_path` attribute?

Reply:
It was an extra attribute that didn't need to be kept around. I wanted to remove as much state as needed so that the `trial.update_storage` would not need to manually set so many properties. (This attribute setting is already not so great..)